cluster/gke/config-test.sh

```diff
@@ -18,11 +18,22 @@
 CLUSTER_NAME="${CLUSTER_NAME:-${USER}-gke-e2e}"
 NETWORK=${KUBE_GKE_NETWORK:-e2e}
 NODE_TAG="k8s-${CLUSTER_NAME}-node"
-IMAGE_TYPE="${KUBE_GKE_IMAGE_TYPE:-container_vm}"
+IMAGE_TYPE="${KUBE_GKE_IMAGE_TYPE:-cos}"
 ENABLE_KUBERNETES_ALPHA="${KUBE_GKE_ENABLE_KUBERNETES_ALPHA:-}"
 
 KUBE_DELETE_NETWORK=${KUBE_DELETE_NETWORK:-true}
 
+# Set e2e test context through NODE_OS_DISTRIBUTION.
+case $(kubectl get node -o=jsonpath='{.items[0].status.nodeInfo.osImage}') in
```

Comment:
Aside from my other comment (that this code will disappear soon), I feel like this code should be moved into `kubetest` anyways - it's a generic check and would be fine to implement across providers after deployment.